server-side/controllers/profile.controller.js

```diff
@@ -4,18 +4,13 @@ import Profiles from '../models/profile.model.js';
 
 export const getAllProfiles = async (req, res, next) => {
     try {
-        const profiles = await Profiles.find().populate('limitedWebsites.websiteId blockedSites').select('-__v');
+        const profiles = await Profiles.find().populate('listWebsites').select('-__v');
```

Comment:
Are the changes in the file related to this task? If not, remove them

Reply:
✅